Sources/MongoSwift/WriteConcern.swift

```diff
@@ -39,89 +36,64 @@ public class WriteConcern: Codable {
     }
 
     /// Indicates the `W` value for this `WriteConcern`.
-    public var w: W? {
-        let number = mongoc_write_concern_get_w(self._writeConcern)
-        switch number {
-        case MONGOC_WRITE_CONCERN_W_DEFAULT:
-            return nil
-        case MONGOC_WRITE_CONCERN_W_MAJORITY:
-            return .majority
-        case MONGOC_WRITE_CONCERN_W_TAG:
-            if let wTag = mongoc_write_concern_get_wtag(self._writeConcern) {
-                return .tag(String(cString: wTag))
-            }
-        default:
-            break
-        }
-
-        return .number(number)
-    }
+    public var w: W?
```

Comment:
If these properties are mutable, it's possible that a user could modify them to get to an invalid write concern, bypassing the checks we have in the initializer. Because of that, let's make `w`, `journal`, and `wtimeoutMS` immutable as they are today, and we can figure out later if we want to change the API to accommodate mutability.

Reply:
@kmahar Making the WriteConcern values immutable will require having the else statements that initialize them to nil in the `init` with `mongoc_write_concern_t`. Swift complains because it thinks if statements could fail and possibly the switch statements too?
`Error: return from initializer without initializing all stored properties`

Reply:
ok we can explicitly set them to nil in those cases then 👍